optlang/tests/test_glpk_exact_interface.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2013 Novo Nordisk Foundation Center for Biosustainability, DTU.
+# See LICENSE for details.
+
+import os
+import random
+
+import nose
+from optlang import glpk_exact_interface
+from optlang.tests import test_glpk_interface
+from optlang.glpk_exact_interface import Variable, Constraint, Model, Objective
+
+random.seed(666)
+TESTMODELPATH = os.path.join(os.path.dirname(__file__), 'data/model.lp')
+TESTMILPMODELPATH = os.path.join(os.path.dirname(__file__), 'data/simple_milp.lp')
+
+test_glpk_interface.Variable = Variable
```

Comment:
This is pretty hacky, and breaks `test_glpk_interface`
I think it would be cleaner to inherit from `abstract_test_cases` and explicitly add any additional tests to this module. Alternatively all the extra test cases in `test_glpk_interface` should be changed to use `self.interface.Variable` instead of just `Variable` etc

Reply:
Done